packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx

```diff
@@ -43,19 +43,23 @@ type GetCommentReactionsArgs = {
   activeMemberId: string | null
   processingCommentReactionId: string | null
 }
+const COMMENT_BOX_ID = 'comment-box'
 
 export const CommentsSection: React.FC<CommentsSectionProps> = ({ disabled, video }) => {
   const [sortCommentsBy, setSortCommentsBy] = useState(COMMENTS_SORT_OPTIONS[0].value)
+  const [openModal, closeModal] = useConfirmationModal()
   const [originalComment, setOriginalComment] = useState<CommentFieldsFragment | null>(null)
   const [showEditHistory, setShowEditHistory] = useState(false)
-  const [openDeleteModal, closeDeleteModal] = useConfirmationModal()
-  const [commentBody, setCommentBody] = useState('')
-  const { id } = useParams()
   const reactionPopoverDismissed = usePersonalDataStore((state) => state.reactionPopoverDismissed)
   const { activeMemberId, activeAccountId, signIn, activeMembership } = useUser()
   const { openSignInDialog } = useDisplaySignInDialog()
   const { isLoadingAsset: isMemberAvatarLoading, url: memberAvatarUrl } = useMemberAvatar(activeMembership)
   const [highlightedComment, setHighlightedComment] = useState<string | null>(null)
+  const { id: videoId } = useParams()
+  // indexed by commentId's
+  const [commentInputTextCollection, setCommentInputTextCollection] = useState(new Map<string, string>())
```

Comment:
You didn't have to change this to map 😅  I was fine with using an object in this case.